Assets/Scripts/Game/DaggerfallBankManager.cs

```diff
@@ -539,11 +533,11 @@ private static TransactionResult BorrowLoan(int amount, int regionIndex)
             TransactionResult result = TransactionResult.NONE;
             if (amount < 100)
                 result = TransactionResult.LOAN_REQUEST_TOO_LOW;
-            else if (amount > CalculateMaxLoan())
+            else if (amount > FormulaHelper.CalculateMaxBankLoan())
```

Comment:
Would it be useful to pass the region index to the formula, or should mods just get the current region from the PlayerGPS?

Reply:
Yes, agree.